pkgs/misc/frescobaldi/default.nix

```diff
@@ -19,6 +19,12 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ pyqtwebengine.wrapQtAppsHook ];
 
+  # Needed because source is fetched from git
+  preBuild = ''
+    make -C i18n
+    make -C linux
```

Comment:
do we want to add -j$NIX_BUILD_CORES here?

Reply:
They run in less than 5 seconds, so I don't think it is worth.